node/src/gql/types/answer.ts

```diff
@@ -29,6 +30,24 @@ export const AnswerType = new GraphQLObjectType({
     transcript: { type: GraphQLString },
     status: { type: GraphQLString },
     media: { type: GraphQLList(AnswerMediaType) },
+    videoUrl: {
+      type: GraphQLString,
+      args: {
+        tag: { type: GraphQLString },
+      },
+      resolve: async function (answer: Answer, args: { tag: string }) {
+        if (!answer.media) {
+          return '';
+        }
+        const media = answer.media.find(
+          (m) => m.type === 'video' && m.tag === (args.tag || 'web')
+        );
+        if (!media) {
+          return '';
+        }
+        return process.env.STATIC_URL_BASE + media.url;
```

Comment:
this looks wrong. You can't assume `STATIC_URL_BASE` does or does not have a trailing `/` (in practice it does not)

you should do something like this instead


```typescript
return URL(media.url, process.env.STATIC_URL_BASE);
```

https://nodejs.org/api/url.html